protos/cri/api.proto

```
@@ -0,0 +1,2117 @@
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// To regenerate api.pb.go run `hack/update-codegen.sh protobindings`
+syntax = "proto3";
+
+package runtime.v1;
```

Buf (1.55.1)

Files with package "runtime.v1" must be within a directory "runtime/v1" relative to root but were in directory "protos/cri".

(PACKAGE_DIRECTORY_MATCH)